### PR TITLE
COMMERCE-6286 Maximum Discount Amount is not being applied

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/discount/test/CommerceDiscountV2Test.java [hunk 1]
```diff
@@ -768,6 +768,82 @@ public void testCommercePercentageDiscounts() throws Exception {
...
+		).and(
+			"A discount with fixed value"
+		).when(
+			"I try to get the final price of the product"
+		).then(
+			"The final price will be calculated taking into consideration " +
+				"the discount"
+		);
+
+		CommerceCatalog catalog =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/discount/test/CommerceDiscountV2Test.java:789]: @riccardo-alberti we should name this `commerceCatalog`.
```suggestion
		CommerceCatalog commerceCatalog =
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by riccardo-alberti** [comment]
ci:forward

**Review by dsitu — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**